airflow_provider_fivetran/hooks/fivetran.py

```diff
@@ -1,23 +1,24 @@
+import json
+import logging
 from time import sleep
-from urllib.parse import urlparse
 
 import requests
 from requests import PreparedRequest, exceptions as requests_exceptions
 from requests.auth import AuthBase
 import pendulum
-import logging
-import json
-import time
-
 
-from airflow import __version__
+# from airflow import __version__
```

Comment:
Is there any need for this?

Reply:
@pgzmnk Question for you.

Reply:
I don't see its use within this file. It isn't a requirement for hooks.